src/aedifix/config.py

```diff
@@ -45,6 +46,14 @@ def __init__(
         """
         super().__init__(manager=manager)
         self._config_file_template = config_file_template.resolve()
+
+        config_file = self.template_file
+        if config_file.suffix == ".in":
+            config_file = config_file.with_suffix(
+                ""
+            )  # removes the trailing .in
```

Comment:
```suggestion
            # remove the trailing .in
            config_file = config_file.with_suffix("") 
```

Just a nit but I'd find this alot more readable